unittests/API/APITest.cpp

```diff
@@ -433,6 +433,23 @@ TEST(HermesWatchTimeLimitTest, WatchTimeLimit) {
   }
 }
 
+TEST(HermesTriggerAsyncTimeoutTest, TriggerAsyncTimeout) {
+  // Some code that loops forever to exercise the async interrupt.
+  const char *forEver = "for (;;){}";
+  uint32_t ShortTimeoutMS = 123;
+  {
+    auto rt = makeHermesRuntime();
```

Comment:
For now, we need to manually enable this.

```suggestion
    auto rt = makeHermesRuntime(hermes::vm::RuntimeConfig::Builder().withAsyncBreakCheckInEval(true).build());
```